api/cm-cli/v1alpha1/crd/cm-cli.open-cluster-management.io_printPolicies.yaml

```diff
@@ -0,0 +1,196 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.6.0
+  creationTimestamp: null
+  name: printpolicies.cm-cli.open-cluster-management.io
+spec:
+  group: cm-cli.open-cluster-management.io
+  names:
+    kind: PrintPolicies
+    listKind: PrintPoliciesList
+    plural: printpolicies
+    singular: printpolicies
+  scope: Namespaced
+  versions:
+  - additionalPrinterColumns:
+    - jsonPath: .spec.policy.Name
```

Comment:
I discovered that the list is sort by default on the field which has a jsonpath .metadata.name, I bet here then the jsonpath .spec.policy.Name must be replaced by .metadata.Name... (Btw, didn't know that jsonpath is not case sensitive...)

Reply:
Since `policy.Name` is set to `metadata.name`, it should be equivalent, but I'll adjust it to be clearer. I think `jsonpath` is case sensitive, but underneath the covers it must recognize the go "Name" attribute. That same path doesn't work with `kubectl`. 🤷

Reply:
it is the kubernetes "printObj" method that looks for a jsonphat = .metadata.name to sort on if the --sort-by is not provided.